flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkV2.java

```diff
@@ -148,10 +159,6 @@ private void testChangeLogs(List<String> equalityFieldColumns,
                               List<List<Record>> expectedRecordsPerCheckpoint) throws Exception {
     DataStream<Row> dataStream = env.addSource(new BoundedTestSource<>(elementsPerCheckpoint), ROW_TYPE_INFO);
 
-    // Shuffle by the equality key, so that different operations of the same key could be wrote in order when
```

Comment:
maybe this unit test demonstrates the original intention that FlinkSink assumes certainly ordering of input stream and it is not the responsibility of the FlinkSink to correct any ordering problem.

Reply:
Unfortunately, we could not assumes FlinkSink will got ordering input stream. Such as when using FlinkSQL, we could not easily to control network shuffle between two opr, and that will require user to have a good understanding of how FlinkSink even FlinkSQL works.